src/ResourceManager/Network/Commands.Network/ExpressRouteCircuit/NewAzureExpressRouteCircuitCommand.cs

```diff
@@ -70,20 +70,35 @@ public class NewAzureExpressRouteCircuitCommand : ExpressRouteCircuitBaseCmdlet
         public string SkuFamily { get; set; }
 
         [Parameter(
-            Mandatory = true,
+            ParameterSetName = "ServiceProvider",
```

Comment:
since you are introducing poaram sets, you need to declare the default param set for the cmdlet.